tools/flang2/flang2exe/llutil.h

```diff
@@ -269,6 +269,14 @@ typedef enum LL_InstrListFlags {
   FAST_MATH_FLAG      = (1 << 4), /**< I_CALL only */
   VOLATILE_FLAG       = (1 << 4), /**< I_LOAD, I_STORE, I_ATOMICRMW,
                                        I_CMPXCHG only */
+  /* These two flags overlap with the next two flags below
+   * which do not seem to be used and hence is OK. The math
```

Comment:
In that case, can we delete `CALL_FUNC_CAST_FLAG` and `CALL_FUNC_INDIRECT_CAST` completely?

Reply:
But they are used across cgmain.cpp and llopt.cpp, modifying that places would go beyond the scope of this PR.

Reply:
In that case, the comment is inaccurate. It would be better to push a commit to get rid of those two flags first, ensuring that they are really not used anywhere, before merging this one.

Reply:
I was looking at the wrong repo! Indeed, these enums aren't used anymore. Removed in the most recent changes.